dkg/pedersen/handler.go

```diff
@@ -513,5 +656,321 @@ func (h *Handler) handleDeal(ctx context.Context, msg types.Deal, out mino.Sende
 		}
 	}
 
+	dela.Logger.Debug().Msgf("%s is certified", h.me)
+
 	return nil
 }
+
+func (h *Handler) announceDkgPublicKey(isCommonNode bool, out mino.Sender, from mino.Address) error {
+	// 6. Send back the public DKG key
+
+	var publicKey kyber.Point
+
+	isOldNode := h.startRes.GetDistKey() != nil
+	isNewNode := !isOldNode
+
+	// if the node is new or a common node it should update its state
+	if isNewNode || isCommonNode {
+		distrKey, err := h.dkg.DistKeyShare()
+		if err != nil {
+			return xerrors.Errorf("failed to get distr key: %v", err)
+		}
+		publicKey = distrKey.Public()
+		// 7. Update the state before sending to acknowledgement to the
```

Comment:
```suggestion
		publicKey = distrKey.Public()
		
		// 7. Update the state before sending to acknowledgement to the
```

Also, does this `7.` numbering still make sense?